hw/ip/rom_ctrl/data/rom_ctrl.hjson

```diff
@@ -84,9 +84,48 @@
     },
   ],
   countermeasures: [
-    { name: "BUS.INTEGRITY",
-      desc: "End-to-end bus integrity scheme."
-    }
+    {
+      name: "FSM.SPARSE",
+      desc: "FSMs are sparsely encoded."
+    },
+    {
+      name: "MEM.SCRAMBLE",
+      desc: "The ROM is scrambled."
+    },
+    {
+      name: "MEM.DIGEST",
+      desc: "A cSHAKE digest is computed of the ROM contents."
+    },
+    {
+      name: "BUS.MUBI",
```

Comment:
Hey @rswarbrick - sorry for the delay!

I think this list looks good. As for the `BUS`, should we maybe introduce another asset called `INTERSIG` or similar if we want to denote that this is a point to point connection?

CC @tjaychen for thoughts

Reply:
Sounds like a good idea. I'm going to merge this now, but we can always follow up on this afterwards.

Reply:
Sorry about that. Let's merge it and amend if needed.